### ISSUE TITLE
Click Behaviour

### ISSUE THREAD

**Issue by hew** [notwaldorf/literally#1]
Cool lib. Took it for a spin and it took me a second to realize that clicks on the left go back a slide, and clicks on the right side of the screen move to the next slide. 

It might be worthwhile to use `e-resize` and `w-resize` cursors when the mouse moves to the left/right hand sides of the screen to give a bit of a visual aid. Just an idea :)

**Comment by notwaldorf** [comment]
@hew yeah, the current clicking is totally annoying. I should probably have actual buttons instead

**Comment by hew** [comment]
Buttons work too! 👉
